### ISSUE TITLE
CSS style applied to two classes causes invalid Elm function names to be generated

### ISSUE THREAD

**Issue by yonigibbs** [monty5811/postcss-elm-tailwind#10]
Hi. I'm looking at adding a loading spinner into an app and found some CSS ones [here](https://loading.io/css/) so tried one of those. To do this I added some CSS which included the following into my CSS file:

```css
.lds-roller div {
    animation: lds-roller 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
    transform-origin: 40px 40px;
}
```

This generated an Elm function called `lds_roller div` (i.e. with a space in the function name).

Is this a bug or is there some way I can configure round this?

**Comment by monty5811** [comment]
That's a bug!

Will get it fixed soon. As a temporary workaround, I think these loaders should work: https://projects.lukehaas.me/css-loaders/

**Comment by yonigibbs** [comment]
Awesome!  Thanks for responding so quickly.  I'll give those a bash!

**Comment by icidasset** [comment]
@monty5811 I think my PR #9 fixes this (see point 3).

**Comment by monty5811** [comment]
Fix just released in `0.6.0`.
